packages/components/src/internal/components/domainproperties/models.tsx

```diff
@@ -1373,6 +1374,9 @@ export class DomainField
         const config = {
             measure: DomainField.defaultValues(DOMAIN_FIELD_MEASURE, dataType),
             dimension: DomainField.defaultValues(DOMAIN_FIELD_DIMENSION, dataType),
+            lookupValidator: dataType === SAMPLE_TYPE ? new PropertyValidator(LOOKUP_VALIDATOR_VALUES) : undefined,
```

Comment:
nit: There are a lot of instances (11?) declared as `new PropertyValidator(LOOKUP_VALIDATOR_VALUES)`. Consider defining a constant.

Reply:
Done.